schematic/store/synapse.py

```diff
@@ -2130,50 +2159,46 @@ def upsertTable(synStore: SynapseStorage, sg: SchemaGenerator, tableToLoad: pd.D
         
 
         Args:
-            tableToLoad: manifest formatted appropriately for the table
-            tableName: name of the table to be uploaded
-            existingTableId: synId of the existing table to be replaced     
-            datasetId: synID of the dataset for the manifest    
-            columnTypeDict: dictionary schema for table columns: type, size, etc
+            sg: SchemaGenerator instance
             
-
         Returns:
            existingTableId: synID of the already existing table that had its metadata replaced
         """            
 
-        username, authtoken = TableOperations._get_schematic_db_creds(synStore)
+        username, authtoken = self._get_schematic_db_creds()
 
-        synConfig = SynapseConfig(username, authtoken, synStore.getDatasetProject(datasetId))
+        synConfig = SynapseConfig(username, authtoken, self.synStore.getDatasetProject(self.datasetId))
         synapseDB = SynapseDatabase(synConfig)
 
         try:
             # Try performing upsert
-            synapseDB.upsert_table_rows(table_name=tableName, data=tableToLoad)
+            synapseDB.upsert_table_rows(table_name=self.tableName, data=self.tableToLoad)
         except(SynapseHTTPError) as ex:
             # If error is raised because Table has old `Uuid` column and not new `Id` column, then handle and re-attempt upload
             if 'Id is not a valid column name or id' in str(ex):
-                TableOperations._update_table_uuid_column(synStore, existingTableId, sg)
-                synapseDB.upsert_table_rows(table_name=tableName, data=tableToLoad)
+                self._update_table_uuid_column(sg)
+                synapseDB.upsert_table_rows(table_name=self.tableName, data=self.tableToLoad)
             # Raise if other error
             else:
                 raise ex
 
-        return existingTableId
-
-    def _update_table_uuid_column(synStore: SynapseStorage, table_id: str, sg: SchemaGenerator,) -> None:
+        return self.existingTableId
+    
+    @profile(sort_by='cumulative', strip_dirs=True)
```

Comment:
@linglp @GiaJordan let me remove this before merge

Reply:
@GiaJordan I just removed this line.

Reply:
Thank you!